docs/mllib-naive-bayes.md

```diff
@@ -115,22 +115,31 @@ used for evaluation and prediction.
 
 Note that the Python API does not yet support model save/load but will in the future.
 
-<!-- TODO: Make Python's example consistent with Scala's and Java's. -->
 {% highlight python %}
-from pyspark.mllib.regression import LabeledPoint
 from pyspark.mllib.classification import NaiveBayes
+from pyspark.mllib.linalg import Vectors
+from pyspark.mllib.regression import LabeledPoint
+
+data = sc.textFile("data/mllib/sample_naive_bayes_data.txt")
+
+# Preprocessing
+splitData = data.map(lambda line: line.split(','))
+parsedData = splitData.map(
```

Comment:
We can define a parse function to make the code more readable. Btw, we use 4 space indentation in Python, following PEP8.

~~~python
def parseLine(line):
    parts = line.split(',')
    label = float(parts[0])
    features = Vector.dense([float(x) for x in parts[1].split(' ')])
    return LabeledPoint(label, features)

data = sc.textFile('data/mllib/sample_naive_bayes_data.txt').map(parseLine)
~~~